middleware.go

```diff
@@ -20,10 +20,10 @@ type Middleware struct {
 }
 
 func (m *Middleware) PrometheusHandler() fiber.Handler {
-	return func(c *fiber.Ctx) {
+	return func(c *fiber.Ctx) error {
 		if c.Path() == m.MetricPath {
 			c.Next()
-			return
+			return nil
```

Comment:
We can return the error like this `return c.Next()` or log error.

Reply:
Thanks for the review!
I have updated the code accordingly.